kymatio/scattering1d/scattering1d.py

```diff
@@ -359,8 +301,8 @@ def meta(self):
         meta : dictionary
             See the documentation for `compute_meta_scattering()`.
         """
-        return Scattering1D.compute_meta_scattering(self.J, self.Q,
```

Comment:
remind me: what is the rule behind this modification?

Reply:
the rule is that, on a multiline statement within parentheses, all lines following the opening parenthesis should be at the same indentation level. So either:
1. put the `m` of  `max_order` at the same level as the `s` of `self.J`
or
2. return to next line right after the opening parenthesis so that `self.J, self.Q, max_order=self.max_order` are a single line. in this case the indentation level can be anything

I opted for 2. because 1. would lead to 80+ characters on the second line

Reply:
:pray: